### PR TITLE
added uninstall in README

### LINKED ISSUE #68: add Uninstall Instructions in README.md
I think there should be a mention in README.md on how to uninstall ovos also

### PR DESCRIPTION
**Pull request by suvanbanerjee**
Closes #68

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,19 @@ EOF
...
+
+To uninstall Open Voice OS run the installer with the `--uninstall` option. even if you dont run the installer with the `--uninstall` option, the installer will check if ovos is installed and will ask you if you want to uninstall it.
+
+```shell
+sh -c "curl -s https://raw.githubusercontent.com/OpenVoiceOS/ovos-installer/main/installer.sh -o installer.sh && chmod +x installer.sh && sudo ./installer.sh -u && rm installer.sh
+```
+or
+``` shell
+sudo ./installer.sh -u
+```
```
Comment (goldyfruit) [inline README.md]: I would remove this part as it might confuse users.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,15 @@ EOF
 
 Few scenarios are available as example in the [scenarios](https://github.com/OpenVoiceOS/ovos-installer/tree/main/scanerios) directory of this repository.
 
+## Uninstall
+
+To uninstall Open Voice OS run the installer with the `--uninstall` option. even if you dont run the installer with the `--uninstall` option, the installer will check if ovos is installed and will ask you if you want to uninstall it.
```
Comment (goldyfruit) [inline README.md]: When using the `-u`/`--uninstall` option, the installer doesn't ask confirmation, it doesn't go at all in the TUI.

File: README.md [hunk 3]
```diff
@@ -59,6 +59,15 @@ EOF
...
+
+```shell
+sh -c "curl -s https://raw.githubusercontent.com/OpenVoiceOS/ovos-installer/main/installer.sh -o installer.sh && chmod +x installer.sh && sudo ./installer.sh -u && rm installer.sh
```
Comment (goldyfruit) [inline README.md]: As you mentioned `--uninstall` option above, this should be used as well in the command instead of `-u`.

**Review by goldyfruit — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Few scenarios are available as example in the [scenarios](https://github.com/Ope
 
 ## Uninstall
 
-To uninstall Open Voice OS run the installer with the `--uninstall` option. even if you dont run the installer with the `--uninstall` option, the installer will check if ovos is installed and will ask you if you want to uninstall it.
+To uninstall Open Voice OS run the installer with the `--uninstall` option.
```
Comment (goldyfruit) [inline README.md]: To uninstall Open Voice OS run the installer with the `--uninstall` option *(non-interactive)* or simply run the installer and answer **"Yes"** to the *"Do you want to uninstall Open Voice OS?"* question.

Comment (suvanbanerjee) [reply README.md]: thanks :smile:

**Review by goldyfruit — APPROVED** [review]

**Comment by goldyfruit** [comment]
Thanks @suvanbanerjee

**Pull request merged**